dbms/src/Storages/DeltaMerge/DeltaMergeStore_Statistics.cpp

```diff
@@ -235,21 +235,15 @@ LocalIndexesStats DeltaMergeStore::getLocalIndexStats()
             const auto & delta = segment->getDelta();
             if (const auto lock = delta->getLock(); lock)
             {
-                const auto & mem_table = delta->getMemTableSet();
-                index_stats.rows_delta_not_indexed += mem_table->getRows();
+                index_stats.rows_delta_not_indexed += delta->getRows();
```

Comment:
why change the logic to first set to delta->getRows, then subtract the rows of not indexed tiny_file rows?

Reply:
reduce virtual function calls